### ISSUE TITLE
Add latest NO2 data OMI

### ISSUE THREAD

**Issue by olafveerman** [NASA-IMPACT/covid-dashboard#283]
We previously processed until March 2020. See if there's new data that we can add to the API.

@rwegener2 @abarciauskas-bgse

**Comment by abarciauskas-bgse** [comment]
I have updated the monthly products to include April 2020. May 2020 is not yet available
https://covid-eo-data.s3.amazonaws.com/OMNO2d_HRM/OMI_trno2_0.10x0.10_202004_Col3_V4.nc.tif

I have also updated the daily product although I think these aren't being used in the dashboard from what I can tell (was only up to May 07, 2020 now is up through June 21, 2020, for example:
https://covid-eo-data.s3.amazonaws.com/OMNO2d_HRD/2020_06_21_NO2TropCS30_Col3_V4.hdf5.cog.tif

If we're using the daily data I am definitely interested in spending more time on using a scheduled task so it updates automatically. If we're just doing the monthly data than doing a monthly manual batch job is not a big lift and I might the process as-is.
